addons/point_of_sale/models/pos_session.py

```diff
@@ -126,12 +126,20 @@ def _compute_cash_all(self):
     @api.constrains('user_id', 'state')
     def _check_unicity(self):
         # open if there is no session in 'opening_control', 'opened', 'closing_control' for one user
-        if self.search_count([('state', 'not in', ('closed', 'closing_control')), ('user_id', '=', self.user_id.id)]) > 1:
```

Comment:
Here we change the unicity constraints to allow > 1 session to be open which is what we want but this requires two more changes to avoid tracebacks on the POS dashboard:

```
Traceback (most recent call last):
...
  File "/home/odoo/odoo/addons/point_of_sale/models/pos_config.py", line 141, in _compute_current_session
    pos_config.current_session_id = session
  File "/home/odoo/odoo/odoo/fields.py", line 887, in __set__
    value = self.convert_to_cache(value, record)
  File "/home/odoo/odoo/odoo/fields.py", line 1902, in convert_to_cache
    raise ValueError("Wrong value for %s: %r" % (self, value))
ValueError: Wrong value for pos.config.current_session_id: pos.session(23, 22)
```

```
Traceback (most recent call last):
...
  File "/home/odoo/odoo/addons/point_of_sale/models/pos_config.py", line 163, in _compute_current_session_user
    pos_config.pos_session_username = pos_config.session_ids.filtered(lambda s: s.state == 'opened').user_id.name
  File "/home/odoo/odoo/odoo/fields.py", line 863, in __get__
    record.ensure_one()
  File "/home/odoo/odoo/odoo/models.py", line 4803, in ensure_one
    raise ValueError("Expected singleton: %s" % self)
ValueError: Expected singleton: pos.session(23, 22)
```

On ```pos.config``` both ```_compute_current_session()``` and ```_compute_current_session_user()``` need to only search on non-rescue sessions. So for ```_compute_current_session()``` something like:

```python
and '(RESCUE FOR ' not in r.name
```

can be added to the ```filtered``` lambda. Similar for ```_compute_current_session_user()```.

This is analogous to how it was in 9.0: [_compute_current_session()](https://github.com/odoo/odoo/blob/9.0/addons/point_of_sale/point_of_sale.py#L48), [_compute_current_session_user()](https://github.com/odoo/odoo/blob/9.0/addons/point_of_sale/point_of_sale.py#L80)